pkg/llmstrategy/react/react_prompt_template_default.txt

```diff
@@ -22,8 +22,8 @@ If you need to use a tool:
     "thought": "Your detailed reasoning about what to do next",
     "action": {
         "name": "Tool name ({{.Tools}})",
-        "reason": "Explanation of why you chose this tool (not more than 100 words)",
-        "input": "complete command to be executed.",
+        "reason": "Explanation of why you chosse this tool (not more than 100 words)",
```

Comment:
choose?

Reply:
Aah... thanks for catching this. fixing it.